test/functional/shada/merging_spec.lua

```diff
@@ -13,6 +13,12 @@ local read_shada_file = shada_helpers.read_shada_file
 local wshada, sdrcmd, shada_fname =
   get_shada_rw('Xtest-functional-shada-merging.shada')
 
+local mock_file_path = '/a/b/'
+local header = '\xc4\x06'
```

Comment:
I do not see where `header` variable changes, so what’s the point in its existence at all?